### ISSUE TITLE
Sealing keys with PCRs

### ISSUE THREAD

**Issue by stv0g** [Foxboron/age-plugin-tpm#21]
It would be nice if we could also seal the keys given a set of PCR values.

Even nicer would be support for [FAPI policies](https://tpm2-software.github.io/fapipolicies/) which could be provided in a JSON file. But I currently see no FAPI support in go-tpm. So its probably quite difficult to implement this without tpm2-tss's [`Fapi_Import()`](https://github.com/tpm2-software/tpm2-tss/blob/d0632dabe8557754705f8d38ffffdafc9f4865d1/include/tss2/tss2_fapi.h#L127)?

**Comment by Foxboron** [comment]
Yes, but for that I'd like a `Reseal` API to function. So I need to learn how this works in Go, along with probably implementing missing apis in `go-tpm`.

See https://github.com/Foxboron/ssh-tpm-agent/issues/15
